### ISSUE TITLE
eeg_ica not finding blink components

### ISSUE THREAD

**Issue by stonekate** [bnicenboim/eeguana#128]
Hey Bruno, as mentioned the other day (week?) I've run eeg_ica and plotted the results using plot_components(), which doesn't show any blink-like components. However, when I do artifact rejection, it finds some obvious blinks. Maybe I'm not setting something right? But just in case something else is going on, I've put an annotated R script and relevant datasets are in Dropbox/eeguana_questions

**Comment by bnicenboim** [comment]
If you do this,
```
eeg_ica_summary_tbl(eegica %>% eeg_filt_band_pass(EOGV, EOGH, freq = c(.1, 30)))
```
You'll notice that ICA9 and ICA10 have high correlations with the eye channels,
then you can do this
```
plot_components(eegica, c("ICA10","ICA9"))
```
and it's clear to me that these are good candidates.
Ploting all the components at the same time might be a bit problematic, the scale is fine, but it's just that the colors are hard to distinguish.

One other thing is that you re-reference to M1-M2, this should be done before the ICA. Maybe this is also a problem.

It seems to me that this is working, if Brainvision/MNE makes a much better job, feel free to open another issue comparing their performance.

In any case, thanks for checking this stuff!
